src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -387,7 +396,10 @@ def list_windows(self):
 
     @keyword
     def unselect_frame(self):
-        """Sets the top frame as the current frame."""
+        """Sets the top frame as the current frame.
+        
+        It does the same as `Select Window` without arguments, returning to the Main frame.
```

Comment:
By reading the code, for me it looks like `Select Window` and unselect frame are not the same things. `Select Window` is able to switch between different windows and unselect frame just returns to the top most frame set.

Reply:
I specified "without arguments" for this case.

Reply:
It is wrong to indicate that they are the same thing, because they are not. Could you remove this update.

Reply:
I don't say they are the same things.
How about saying?

> It results in the same effect as `Select Window` without arguments, returning to the Main frame.

Reply:
I think the current documentation is good. Why we should tell about Select Window keyword.